src/ast/codegen_llvm.cpp

```diff
@@ -1313,7 +1313,9 @@ void CodegenLLVM::visit(AssignMapStatement &assignment)
   Value *val, *expr;
   expr = expr_;
   AllocaInst *key = getMapKey(map);
-  if (map.type.type == Type::string || assignment.expr->type.type == Type::inet)
+  if (map.type.type == Type::string ||
+      assignment.expr->type.type == Type::inet ||
+      assignment.expr->type.type == Type::usym)
```

Comment:
Just to double check: this is ok b/c the usym is already on the stack, right?

Reply:
Yes.

Found a different bug though  `:f { @=usym(arg0); @a=@ }'` results in a segv. I've added a commit to fix it :)